fpna/Tests/ExtensionMethodsTest.ipynb

```diff
@@ -0,0 +1,165 @@
+{
+    "metadata": {
+        "authors": [],
+        "id": "0Sv-iuexO02G_9VRlhVTLw",
+        "kernelspec": {
+            "display_name": "Formula Framework",
+            "language": "C#",
+            "name": "C#"
+        },
+        "language_info": {
+            "file_extension": ".cs",
+            "mimetype": "text/plain",
+            "name": "C#"
+        }
+    },
+    "nbformat": 4,
+    "nbformat_minor": 5,
+    "cells": [
+        {
+            "cell_type": "markdown",
+            "source": [
+                "<p style=\"font-weight:bold;\"> <span style=\"font-size: 36px\"> Extension Methods Test </span> </p>"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "code",
+            "source": [
+                "#!import \"../CalculationEngine\""
```

Comment:
I think you only need the ExtensionMethods

Reply:
I am using "Precision" that is defined in ifrs CE so I change this to point there now.